.eslintrc.json

```diff
@@ -330,9 +342,15 @@
     "no-unreachable": "error",
     "no-unsafe-finally": "error",
     "no-unsafe-negation": "error",
-    "no-restricted-imports": ["error", {
-      "patterns": ["original-works-core/src", "original-works-core/dist"]
-    }],
+    "no-restricted-imports": [
+      "error",
+      {
+        "patterns": [
+          "original-works-core/src",
+          "original-works-core/dist"
+        ]
```

Comment:
eee... are you sure it is from this project?

Reply:
hahaha, if you look closely All I did here is reformat, so that was here before. Not sure how it got here.